index.js

```diff
@@ -163,6 +166,7 @@ function Xray() {
     }
 
     function load(html, url) {
+      html = html || "";
```

Comment:
Possibly a little redundant given the particular case I am handling, but I suspect it is not the only condition under which a null reference exception is provoked here. See #84.